section2-week1/exercise6/Main.cpp

```diff
@@ -0,0 +1,37 @@
+/*Exercise 6 - Multiple Choice
+Write a program where the user can make a choice out of a list of options. Make sure to use an enum and handle when the input is out of bounds.
+*/
+
+#include <iostream>
+using namespace std;
+
+int main()
+{
+	cout << "WELCOME PLAYER, choose your character" << endl;
+	cout << "1 - Gardian\n2 - Wizard\n3 - Warrior\n4 - Healer" << endl;
+
+	int cho;
```

Comment:
What is cho supposed to mean? Is it choice? Is it really necessary to abbreviate choice?

Reply:
Remember to think about the reader when you write your code. What will make sense and be helpful for someone else reading your code?

Reply:
thanks! i'll take on mind :3